crates/db_connection_pool/src/postgrespool.rs

```diff
@@ -157,7 +159,11 @@ impl PostgresConnectionPool {
                 if let Host::Tcp(host) = host {
                     verify_ns_lookup_and_tcp_connect(host, *port)
                         .await
-                        .context(InvalidHostOrPortSnafu)?;
+                        .map_err(|err| {
+                            tracing::debug!("Failed to connect to host: {}", err);
```

Comment:
Do we need tracing here? it doesn't print any log and the InvalidHostOrPortSnafu will already show the error in user logs

Reply:
you're right, it is useless here, removed